src/App/App.js

```diff
@@ -60,15 +61,14 @@ class App extends React.Component {
     this.removeListener();
   }
 
-  //  isAuthenticated = () => {
-  //    this.setState({ authed: true });
-  //  }
-
   render() {
-    const { authed, pendingUser, weather } = this.state;
+    const { authed, pendingUser, uid } = this.state;
```

Comment:
you are currently not using your `authed` variable within the context of this render function. Please either remove it from the object destructuring assignment here or change the references to `this.state.authed` in your Navbar and Routes to `authed`.